liquibase-couchbase/src/test/java/integration/statement/MutateInStatementIT.java

```diff
@@ -53,8 +53,8 @@ void Should_insert_property_by_provided_path_for_specific_docId() {
 
         new MutateInStatement(mutate, mutateInOptions().timeout(Duration.ofSeconds(2))).execute(clusterOperator);
         Collection collection = collectionOperator.getCollection();
-        assertThat(collection).extractingDocument(doc2.getId()).hasNoField("age");
-        assertThat(collection).extractingDocument(doc1.getId()).hasField("age");
+        assertThat(collection).extractingDocument(doc2.getId()).jsonHasNoField("age");
```

Comment:
Maybe we could check by the separate method that the document content is Json and then - hasField/hasNoField? Not sure renaming is meaningful, only Json content could have fields/values in our case

Reply:
Done